### ISSUE TITLE
Logits scores of beam searched reply variants not available

### ISSUE THREAD

**Issue by bigdatabaracus** [pbhatia243/Neural_Conversation_Models#8]
Hi, I'm trying to get access to the logit scores of the reply variants produced by the beam search. I was expecting the variable 'output_logits' produced by the model.step function in neural_conversation_model.py ( https://github.com/pbhatia243/Neural_Conversation_Models/blob/master/neural_conversation_model.py#L253 ) to contain this information, but this doesn't seem to be the case in the block where beamsearch is turned on.
